contracts/validator-manager/ValidatorManager.sol

```diff
@@ -62,18 +62,24 @@ abstract contract ValidatorManager is Initializable, ContextUpgradeable, IValida
     uint8 public constant BLS_PUBLIC_KEY_LENGTH = 48;
     bytes32 public constant P_CHAIN_BLOCKCHAIN_ID = bytes32(0);
 
-    error MaxChurnRateExceeded();
-    error InvalidWarpMessage();
-    error InvalidBlockchainID();
-    error InvalidAddress();
-    error UnexpectedRegistrationStatus();
-    error InvalidValidatorStatus();
-    error InvalidValidationID();
-    error NodeAlreadyRegistered();
-    error InvalidSubnetConversionID();
-    error InvalidInput();
-    error InvalidExpiry();
+    error InvalidValidatorManagerAddress(address validatorManagerAddress);
+    error InvalidWarpOriginSenderAddress(address senderAddress);
+    error InvalidValidatorManagerBlockchainID(bytes32 blockchainID);
+    error InvalidWarpSourceChainID(bytes32 sourceChainID);
+    error InvalidRegistrationExpiry(uint64 registrationExpiry);
     error InvalidInitializationStatus();
```

Comment:
tiny nit feel free to ignore but it doesn't seem unreasonable to me to also give this a `bool` parameter. to distinguish between validator already initialized or not

Reply:
I debated that, but seeing as invalid is only thrown when the bool paramter is false, seemed not needed. Not opposed to it either.